build-calico-resource.sh

```diff
@@ -47,16 +47,13 @@ function fetch_and_validate() {
   fi
 }
 
-wget \
-  https://github.com/projectcalico/calico/releases/download/$calico_version/release-$calico_version.tgz
-tar -xf release-$calico_version.tgz
-
 for arch in ${arches}; do
   rm -rf resource-build-$arch
   mkdir resource-build-$arch
   pushd resource-build-$arch
-  cp ../release-$calico_version/bin/calicoctl/calicoctl-linux-$arch calicoctl
-
+  fetch_and_validate \
+    https://github.com/projectcalico/calico/releases/download/$calico_version/calicoctl-linux-$arch
+  mv calicoctl-linux-$arch calicoctl
```

Comment:
rather than copy those two files from the extracted tarball -- just download the calicoctl that we need for that arch